frontend/views/components/ProfileCard.vue

```diff
@@ -0,0 +1,241 @@
+<template lang='pug'>
+tooltip(
+  :direction='direction'
+  :manual='true'
+  ref='tooltip'
+  :opacity='1'
+)
+  slot
+
+  template(slot='tooltip')
+    .card.c-profile(v-if='profile')
```

Comment:
One major requirement to create an effective trapfocus system is to do it within a "dialog" element. As done in modal, we need to do it here too: `role="dialog"` [role dialog MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role) and it needs an accessible label. Here it can be `aria-label="{username} profile"`